C/Chafa/build_tarballs.jl

```diff
@@ -0,0 +1,30 @@
+using BinaryBuilder
+
+name = "Chafa"
+version = v"1.4.1"
+
+sources = [
+    ArchiveSource("https://hpjansson.org/chafa/releases/chafa-$(version).tar.xz",
+                  "46d34034f4c96d120e0639f87a26590427cc29e95fe5489e903a48ec96402ba3"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/chafa-*/
+./autogen.sh
```

Comment:
```suggestion
./autogen.sh --prefix=$prefix --build=${MACHTYPE} --host=${target}
```
I guess?

Reply:
Ah, I got the prefix and host but not `MACHTYPE`; what is that?

Reply:
`${MACHTYPE}` is the triplet of the build environment, so `x86_64-linux-musl` everywhere.  Sometimes build systems explicitly ask for it